signatureverifier/src/test/kotlin/io/github/rakutentech/signatureverifier/api/PublicKeyFetcherSpec.kt

```diff
@@ -98,43 +115,43 @@ class PublicKeyFetcherErrorSpec : PublicKeyFetcherSpec() {
         fetcher.fetch("test_key_id")
     }
 
-    @Test(expected = Exception::class)
-    fun `should throw when the 'id' key is missing in response`() {
+    @Test
+    fun `should return valid key even if 'id' key is missing in response`() {
```

Comment:
Is `id` marked as optional in the endpoint specification?

Reply:
Will change this on a separate PR.